src/content/tests.py

```diff
@@ -1,3 +1,30 @@
-from django.test import TestCase
+from django.urls import reverse
+from django.contrib.auth.models import User
+from django.test import Client, TestCase
 
-# Create your tests here.
+
+class ContentViewTest(TestCase):
+    def setUp(self):
+        self.c = Client()
+        self.c.force_login(User.objects.get_or_create(username="testuser")[0])
```

Comment:
For the index page we don't need to have a logged in user, you can remove this part.

Reply:
thanks for the inputs, I followed up with a commit